src/genericComponents/Input.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { FormControl, InputGroup } from 'react-bootstrap';
+import styled from 'styled-components';
+
+const InputField = (props: any) => {
+  const { placeholder } = props;
+
+  return (
+    <CAGInput>
+      <InputGroup className="mb-3">
+        <FormControl
+          aria-describedby="basic-addon1"
```

Comment:
The point of having a generic component like this is to account for a wide variety of potential cases. For example, we may have helper text, or an icon, or a different size, a label or no label, readonly, etc. And the names, labels, placeholder text, etc. will all be different depending on context, so we need to create props for them. 

Consider this example:
```
  <Form.Group controlId="formBasicEmail">
    <Form.Label>Email address</Form.Label>
    <Form.Control type="email" placeholder="Enter email" />
    <Form.Text className="text-muted">
      We'll never share your email with anyone else.
    </Form.Text>
  </Form.Group>
```
Just to account for this one possibility, we need to think about the following props:
`controlId, label, inputType, placeholder, helperTextClasses, helperText`
But there are instances where these won't be needed, or that we want to default to something (for example, we could default `inputType` to `text`).